spec/features/purl_parser_spec.rb

```diff
@@ -51,4 +51,13 @@
       expect(purl.exists?).to be_falsey
     end
   end
+  describe '#modified_time' do
+    let(:sample_doc_path) do
+      DruidTools::PurlDruid.new('bb050dj7711', purl_fixture_path).path
+    end
+    subject { described_class.new(sample_doc_path) }
+    it 'gets the file modified_time from file system' do
+      expect(subject.modified_time).to be_an Time
+    end
```

Comment:
we should verify that the `modified_time` is coming from the file system. something like this maybe:

```
before_modified = Time.now
FileUtils.touch(sample_doc_path)
expect(subject.modified_time).to be > before_modified
```